app/src/apps/Examinations/lib/Examination.js

```diff
@@ -26,6 +26,11 @@ class Examination {
   submit() {
     this.submitted = true;
   }
+
+  // Change state of examination to submitted
+  submit() {
+    this.submitted = true;
+  }
```

Comment:
I don't think we need a duplicate submit method

Reply:
For sure! I think it was left accidentally from the previous pull request. I will remove it.